src/main/java/com/github/lmiljak/dowertefence/commands/DTCommand.java

```diff
@@ -0,0 +1,25 @@
+package com.github.lmiljak.dowertefence.commands;
+
+import nl.taico.taeirlib.commands.CommandArgs;
+import nl.taico.taeirlib.commands.TCommand;
+
+/**
+ * Class for handling the /dt command and subcommands.
+ */
```

Comment:
The cmd framework doesn't care. It simply performs a reverse lookup amongst all registered command handlers of the most applicable handler for the given command.